File: Equal Distribution.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c;
        scanf("%d%d",&a,&b);
        c=a+b;
        if(c%2==0)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: Equal Distribution.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a,b,s;
            a=sc.nextInt();
            b=sc.nextInt();
            s=a+b;
            if(s%2==0)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}